run.py

```diff
@@ -19,13 +19,19 @@
     parser = argparse.ArgumentParser()
     parser.add_argument('--lemmatization', help='Perform lemmatization (using spaCy)', action='store_true')
     parser.add_argument('--train', help='Perform training', action='store_true')
-    parser.add_argument('--goal', help='Goal of the classifier', choices=['bug', 'regression', 'tracking', 'qaneeded', 'uplift', 'component', 'devdocneeded'], default='bug')
+    parser.add_argument('--goal',
+                        help='Goal of the classifier',
+                        choices=['bug', 'regression', 'tracking', 'qaneeded', 'uplift', 'component', 'devdocneeded', 'triage'],
+                        default='bug')
     parser.add_argument('--classify', help='Perform evaluation', action='store_true')
     parser.add_argument('--generate-sheet', help='Perform evaluation on bugs from last week and generate a csv file', action='store_true')
     args = parser.parse_args()
 
     model_file_name = f'{args.goal}model'
 
+    probabilities = True
```

Comment:
Nit: let's call these `probabilities_enabled` and `importances_enabled`, so you don't have to rename the pre-existing `importances` variable